init.lua

```diff
@@ -41,11 +42,37 @@ local function re_run_failed_workflow(full_repo, run_id)
     log.info('Api response body is '..res.body)
 end
 
+--- Fill table from string input
+local function fill_table(inputString, resultTable)
+    for i in string.gmatch(inputString, "(%w+)") do
+        table.insert(resultTable, i)
+    end
+    return resultTable
+end
+
+--- Check if some table has some expected value
+local function check_value(table, value)
+    for k, v in ipairs(table) do
+        print(v)
```

Comment:
Better to add info string to print, i.e. "print('Value in table: '..v)"

Reply:
It was to debug and see how it works. It is not necessary print. Fixed!